src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -144,6 +149,19 @@ function _interleaveMessagesWithDividers(
         {message.content}
       </MessageMetadata>,
     );
+    if (message.attributes?.displayAlertMessageAfter === "guardian_created_account") {
+      messagesWithDividers.push(
+        <FlexBox
+          className={cssClasses.ALERT_MESSAGE_CONTAINER}
+          key={`alertMessage-${message.index}`}
+        >
+          <FontAwesome name="check" size="lg" />
+          <FlexItem className={cssClasses.ALERT_MESSAGE} grow>
+            Invite accepted! This guardian can now reply to your messages!
```

Comment:
It seems like this context belongs in the consumer (launchpad in this case) of `MessagingThreadHistory` because it is a specific implementation detail. I think the Dewey components are meant to be flexible and contextually-unaware as possible.

Instead I would think that `ThreadHistory` might have another `MessageMetadata` with `type="alert"`. That way we can reuse this alert container for different alerts as they grow at the consumer level. And we could also display different copy based on the user type but that would be up to the consumer.

Also, doesn't this mean that guardians will also see this message? Or is that what is supposed to happen

Reply:
Good points! Refactored it so that more of the logic is left to the client, which should address both issues. Opted to just create a separate component from MessageMetadata since the component is quite different.